docs/usage/key-concepts/dashboard.md

```diff
@@ -49,6 +49,19 @@ To disable the Dependency Dashboard, add the preset `:disableDependencyDashboard
 
 This section explains some common use cases where having the Dependency Dashboard can help.
 
+### Warnings for deprecated dependencies
+
+If deprecated packages are found - or any packages have community-sourced replacement PRs available - then Renovate will add a prominent warning about these packages near the top of the Dependency Dashboard.
+Here is an example of how this can look:
+
+> [!WARNING]
+> The following dependencies are deprecated:
+
+| Datasource | Name                | Replacement?                                                                      |
+| ---------- | ------------------- | --------------------------------------------------------------------------------- |
+| npm        | `airbnb-prop-types` | ![Available](https://img.shields.io/badge/available-green?style=flat-square)      |
```

Comment:
it'd be cool if the "available" badge was a link to the npm package page of the replacement.

Reply:
My first preference was to link to the _open PR_ if one existed, as that would be even more valuable. But I didn't want perfect to be the enemy of good here, and we can enhance this later - I've created an issue for this: https://github.com/renovatebot/renovate/issues/29696